xslt/mesh-rdf-prefixes.ent.template

```diff
@@ -1,4 +1,4 @@
-<!ENTITY mesh "http://id.nlm.nih.gov/mesh/">
+<!ENTITY mesh "http://id.nlm.nih.gov/MESHRDF_PATH/">
 <!ENTITY meshv "http://id.nlm.nih.gov/mesh/vocab#">
```

Comment:
I think both `mesh:` and `meshv:` URIs will either not have the year in them ("current") or have the year in them. You only have `mesh:` parameterized here.